src/main/java/io/nats/client/NUID.java

```diff
@@ -98,15 +99,15 @@ public NUID() {
      * @return the next NUID string from a shared global NUID instance
      */
     public static synchronized String nextGlobal() {
-        return getInstance().next();
+        return StandardCharsets.US_ASCII.decode(getInstance().next()).toString();
```

Comment:
Why the decode?

Reply:
That was since `getInstance().next()` was returning a `ByteBuffer`, I've refactored it to return a `String` now.